app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php

```diff
@@ -58,12 +55,15 @@ public function processWebhookData()
         $collection->addFieldToFilter('processed', array('eq' => 0));
         $collection->getSelect()->limit(self::BATCH_LIMIT);
 
+        $this->_loadGroups();
```

Comment:
This function should be called in construct since groups should be loaded every time this class is used.